src/context/cart-context.tsx

```diff
@@ -40,12 +47,25 @@ export function CartContextProvider(props: any){
 		}, 2000);
 	}
 
+	//Getting total price for all items on cart
+	const getTotalHandler = () => {
+		let totalValue = 0;
+		currentItems.forEach((item: CartItem) => {
+			totalValue = totalValue + item['total'];
+		})
+		//Round on 2 decimals
+		const roundedValue = totalValue.toFixed(2);
+		setCurrentTotal(parseFloat(roundedValue));
```

Comment:
no need for declaring `roundedValue`, you can use `totalValue.toFixed(2)` directly in `setCurrentTotal`